tests/kafkatest/version.py

```diff
@@ -119,7 +119,7 @@ def get_version(node=None):
         return DEV_BRANCH
 
 DEV_BRANCH = KafkaVersion("dev")
-DEV_VERSION = KafkaVersion("3.4.0-SNAPSHOT")
+DEV_VERSION = KafkaVersion("3.5.0-SNAPSHOT")
 
 LATEST_METADATA_VERSION = "3.3"
```

Comment:
@cmccabe can you just confirm whether this is correct & hasn't been bumped in 3.5?